mrp_client.py

```diff
@@ -22,33 +26,189 @@ def getnetmask(self, machine_name, interface_name='eth1'):
         ipgetter = IPGetter(self.url, self.token, machine_name, interface_name)
         return ipgetter.get_netmask()
 
+    def provision(self, machine_name, preseed_name, initrd_desc, kernel_desc,
+                  kernel_opts, arch, subarch):
+        try:
+            provisioner = MachineProvisioner(
+                self.url, self.token, machine_name)
+            provisioner.set_machine_parameters(preseed_name, initrd_desc,
+                                               kernel_desc, kernel_opts, arch,
+                                               subarch)
+            provisioner.machine_provision()
+            return True
+        except ProvisionerError as err:
+            print(err)
+
+    def setparams(self, machine_name, preseed_name, initrd_desc, kernel_desc,
+                  kernel_opts, arch, subarch):
+        try:
+            provisioner = MachineProvisioner(
+                self.url, self.token, machine_name)
+            provisioner.set_machine_parameters(preseed_name, initrd_desc,
+                                               kernel_desc, kernel_opts, arch,
+                                               subarch)
+        except ProvisionerError as err:
+            print(err)
```

Comment:
There is no consistent error control. 
- Some return True, others don't
- None return false on error
- Print goes to stdout, not stderr
- No stack trace, no error handling other than printing it